examples/my-indexer-client/python-client/my_indexer/client.py

```
@@ -0,0 +1,131 @@
+        
+        try:
+            response = requests.request(
+                method=method,
+                url=url,
+                json=json,
+                params=params,
+            )
+            response.raise_for_status()
+            return response.json()
+        except requests.ConnectionError as e:
+            raise IndexerConnectionError(f"Failed to connect to indexer: {e}")
```

Ruff (0.8.0)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)